### PR TITLE
Improve shard allocation awareness documentation - part 2

### PR DESCRIPTION
**Pull request by patelsmit32123**
### Description
Shard allocation awareness attributes documentation was lacking information regarding how the allocation actually functions and in what conditions the shard can remain unallocated.

### Issues Resolved
Closes #[_insert issue number_]

### Version
_List the OpenSearch version to which this PR applies, e.g. 2.14, 2.12--2.14, or all._

### Frontend features
_If you're submitting documentation for an OpenSearch Dashboards feature, add a video that shows how a user will interact with the UI step by step. A voiceover is optional._ 

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by patelsmit32123** [comment]
@Naarcha-AWS I wanted to suggest some restructuring of the changes that we made in last PR. Please take a look.

File: _tuning-your-cluster/index.md [hunk 1]
```diff
@@ -196,19 +196,20 @@ To further fine-tune your shard allocation, you can set custom node attributes f
 
 ### Shard allocation awareness
 
-You can set custom node attributes on OpenSearch nodes to be used for shard allocation awareness. For example, you can set the `zone` attribute on each node to represent the zone in which the node is located. You can also use the `zone` attribute to ensure that the primary shard and its replica shards are allocated in a balanced manner across available, distinct zones. For example, maximum shard copies per zone would equal `ceil (number_of_shard_copies/number_of_distinct_zones)`.
+You can set custom node attributes on OpenSearch nodes to be used for shard allocation awareness. For example, you can set the `zone` attribute on each node to represent the zone in which the node is located. You can also use the `zone` attribute to ensure that the primary shard and its replica shards are allocated in a balanced manner across available, distinct zones. So maximum shard copies per zone would equal `ceil (number_of_shard_copies/number_of_distinct_zones)`.
```
Comment (Naarcha-AWS) [inline _tuning-your-cluster/index.md:199]:
```suggestion
You can set custom node attributes on OpenSearch nodes to be used for shard allocation awareness. For example, you can set the `zone` attribute on each node to represent the zone in which the node is located. You can also use the `zone` attribute to ensure that the primary shard and its replica shards are allocated in a balanced manner across available, distinct zones. In this scenario, maximum shard copies per zone would equal `ceil (number_of_shard_copies/number_of_distinct_zones)`.
```

**Review by Naarcha-AWS — APPROVED** [review]

File: _tuning-your-cluster/index.md [hunk 2]
```diff
@@ -196,19 +196,20 @@ To further fine-tune your shard allocation, you can set custom node attributes f
...
 
-Shard allocation awareness attempts to separate primary and replica shards across multiple zones because 2 shard copies cannot be placed on the same node. When only 1 zone is available, such as after a zone failure, OpenSearch allocates replica shards to the only remaining zone. For example, if your index has a total of 5 shard copies (1 primary and 4 replicas) and nodes in 3 distinct zones, then OpenSearch will perform the following to allocate all 5 shard copies: 
+OpenSearch, by default, allocates shard copies of a single shard across different nodes. When only 1 zone is available, such as after a zone failure, OpenSearch allocates replica shards to the only remaining zone i.e. it considers only available zones (attribute values) for calculating maximum allowed shard copies per zone.
```
Comment (natebower) [inline _tuning-your-cluster/index.md:201]:
```suggestion
OpenSearch, by default, allocates shard copies of a single shard across different nodes. When only 1 zone is available, such as after a zone failure, OpenSearch allocates replica shards to the only remaining zone---it considers only available zones (attribute values) when calculating the maximum number of allowed shard copies per zone.
```

File: _tuning-your-cluster/index.md [hunk 3]
```diff
@@ -196,19 +196,20 @@ To further fine-tune your shard allocation, you can set custom node attributes f
...
+For example, if your index has a total of 5 shard copies (1 primary and 4 replicas) and nodes in 3 distinct zones, then OpenSearch will perform the following to allocate all 5 shard copies:
 
-- Allocate fewer than 2 shards per zone, which will require at least 2 nodes in 2 zones.
+- Allocate not more than 2 shards per zone, which will require at least 2 nodes in 2 zones.
```
Comment (natebower) [inline _tuning-your-cluster/index.md:204]:
```suggestion
- Allocate no more than 2 shards per zone, which will require at least 2 nodes in 2 zones.
```

**Review by natebower — COMMENTED** [review]
@Naarcha-AWS A few changes, please. Thanks!

**Pull request merged**